sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/command_component.py

```diff
@@ -33,43 +33,65 @@
 
 
 class CommandComponent(Component, ParameterizedCommand, AdditionalIncludesMixin):
-    """Command component version, used to define a command component.
+    """Command component version, used to define a Command Component or Job.
 
-    :param name: Name of the component.
+    :param name: The name of the Command job or component.
     :type name: str
-    :param version: Version of the component.
+    :param version: The version of the Command job or component.
     :type version: str
-    :param description: Description of the component.
+    :param description: The description of the component.
     :type description: str
     :param tags: Tag dictionary. Tags can be added, removed, and updated.
     :type tags: dict
-    :param display_name: Display name of the component.
+    :param display_name: The display name of the component.
     :type display_name: str
-    :param command: Command to be executed in component.
+    :param command: The command to be executed during training.
```

Comment:
Why was the wording changed to 'during training'? I thought command components were more general purpose than that.

Reply:
That was my perception as well before beginning this work, but the existing documentation (other than this particular description) regularly aligns Command Job with training. For example, look below at the description for the distribution parameter ("Distribution configuration for distributed training") and these existing documentation descriptions for Command-related entities which all call out training.

- "command", "environment", and "identity" descriptions for Command [here](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.command?view=azure-python) and CommandJob [here](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.commandjob?view=azure-python)

I'm open to removing those references if you think it's more representative of what can be done with a Command. We do control Command-related entities now, so it's within our description. What do you think?

Reply:
Discussed offline - training references have been removed in favor of generic language